src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -45,18 +45,18 @@ public class MvcRazorHost : RazorEngineHost, IMvcRazorHost
         /// </summary>
         /// <param name="root">The path to the application base.</param>
         public MvcRazorHost(string root) :
```

Comment:
Views do not come from webroot, they come from app root. See https://github.com/aspnet/Mvc/blob/448ac5a0dc0bc8c53acd4f74df7c533fec6860d7/src/Microsoft.AspNet.Mvc/RazorViewEngineOptionsSetup.cs#L29